snappi/data-plane-performance/configs/unidirectional.json

```diff
@@ -52,15 +52,41 @@
                     "ethernet": {
                         "dst": {
                             "choice": "value",
-                            "value": "00:0c:29:16:b4:61"
+                            "value": "Agent2Eth1MacAddress"
```

Comment:
Isn't this making it specific to AWS only? Same comment for all the changes made in this file for AWS specific requirements. Can this test still run standalone?